### ISSUE TITLE
Regarding result.json and modifier.csv

### ISSUE THREAD

**Issue by Holorei** [MohamedAliRashad/FreeFaceMoCap#10]
These files are required to run landmarkTransform function, would you briefly explain what kind of data they contain, please? I had a rigged head mesh and once I was using landmarkTransform function from generated rig file, it deforms the mesh in a weird way, I suppose the json file is related to the default rig armature? But I could tell the correlation between them. Thanks in advance.
![image](https://user-images.githubusercontent.com/45619235/170245008-f6d5fb17-c1cc-48b4-9eea-bc2c3b66d7e0.png)
![image](https://user-images.githubusercontent.com/45619235/170245046-2766ec97-76a6-4664-95a2-39f34e591416.png)

**Comment by MohamedAliRashad** [comment]
They contain the transformations from Mediapipe to your rigged points

**Comment by Holorei** [comment]
@MohamedAliRashad  Yep, sure thing. Here is  [google drive](https://drive.google.com/drive/folders/1O2n-udb5vv9txirHY12iiqnRGmkMxfnD?usp=sharing) link for all the related file. 
To reproduce the bug, select the RIG_001 in top right panel, then run the faceCapture1 script.
